### PR TITLE
Revenant-Class Heavy Tug

### PR DESCRIPTION
**Pull request by thearbiber**
## About The Pull Request

A Hardliner scrapping ship, it is about medium-size with a very light armory, crew composition of 8, and monumental slop bins for the gruel enjoyers.

image

image

## Why It's Good For The Game

Currently, the Hardliners are the only playable faction without a proper ship. I am adding this and removing the Komodo to give the faction a fresh coat of paint. Hopefully this new styling will do work to dispel the less than stellar reputation that they've garnered in game. 

## Changelog

:cl:
add: Added new things

/:cl:

### CODE REVIEW

**Comment by Apogee-dev** [comment]
I'll just say what I commented on the dev thread for posterity, but I do feel like the design language here isn't _quite_ what I'd like to see for cybersun-built ships. 

other than that, i've got a strong preference for the numbers-and-letters ship class names i mentioned elsewhere on the discord (stuff like "400j class" or "G52 class") though the ghost names kinda work. i feel like the numbers stand out better from other factions. something to consider if you're not set on revenant for the class name

**Comment by miniusAreas** [comment]
whatever. go my scarab.

**Comment by Madtankdriver** [comment]
H! Just a small suggestion. Looking at the Revenant-Class it doesn't really have much of a good reason to play on this shuttle since the Banshee-class class shuttle and the Phantom-Class would be better picks. So maybe adding an engineering Protolathe or adding more engineering equipment might not be a bad idea. I could definitely see the Revenant-Class being tasked from Cybersun to construct a Blacksite on an ice planet, so in order to do so they would need to work with the Phantom class shuttle for armed support and for refining ore. Alongside the two they would be perfect to construct structures and work together. Take this with a grain of salt, this is what I thought of that might be a good idea to add onto this shuttle to be a tad more useful and good to work with an allied shuttle. Overall, the shuttles look great! Keep up the good work!

**Comment by thearbiber** [comment]
> So maybe adding an engineering Protolathe or adding more engineering equipment might not be a bad idea.

I would sooner be dead than contribute a ship with RnD on it in 2024

File: _maps/configs/syndicate_gorlex_revenant.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	],
+	"map_name": "Revenant Placeholder",
+	"map_short_name": "Revenant-class",
+	"description": "Not finished.",
+	"tags": [
+		"RP Focus",
+		"Combat",
+		"Engineering"
+	],
+	"map_path": "_maps/shuttles/syndicate/syndicate_gorlex_revenant.dmm",
```
Comment (Sadhorizon) [inline _maps/configs/syndicate_gorlex_revenant.json]: Hey, rename it to `syndicate_hardliner_revenant.dmm`. The "gorlex" part is outdated, and is being slowly removed on the NGR side of things.

**Comment by thearbiber** [comment]
updated images, changed ship shape very slightly.

**Comment by thearbiber** [comment]
Should be ready for review I Think.

**Review by Gristlebee — CHANGES_REQUESTED** [review]
- You can set the reinforced tiles under the blast doors in the cargo bay to interior subtype since it starts sealed. Space wind happens otherwise the first time you open it.
image
- Giving the wrecker's gear lockers a unique name or changing the appearance to miners locker would look good instead of "closet".  Access restricting the closets would be good as well since the supply bay isn't access locked. 
image
- The wreckers could also use welding protection of some kind for the angle grinder/cutter. Mining tools of some kind would also be useful. It doesn't look like you have picks or anything of that nature mapped in.
- Use obj/melee/knife/kitchen instead of the base type. The name gets a bit wonky otherwise.
image
image
- Some privacy shutters for the armory desk would be good due to it's proximity to the cargo bay.
image
- You'll want to set the access on the armory windoors to 3 (armory) so only the sergeant is able to access it. 
image
image
- The ship probably doesnt need two sets of welding fuel and water tanks. Consider swapping one of them out for something else.
image
image
- The ship looks very sterile. You could map in some dust and dirt around the ship, some rust on the floors in the engine bay to make it look more lived in.
- The dorms feels a bit sparse. Some decals and linings to separate the lockers/dresser area off from the walkway could look nice.
image
- You'll want to set the access to 19 (heads) on the bridge unless you want the trooper walking in on the bridge.
image
- I think some warning lines between the catwalks and the floors around the engines would make them pop and stand out more as a division.
image
image
image

**Comment by thearbiber** [comment]
Requested changes made.

**Comment by Gristlebee** [comment]
Looks alright to me.

**Review by Latentish — CHANGES_REQUESTED** [review]
The ship so far is looking pretty good, however the  large issue I see with it is the lack of clutter and abundance of open, unused space.
Take this room for example.
image
It has good space allocation, markings, and general vibe, however the room is very very empty. Remember that players can decon simple structures that you add in for clutter, which is what your ship needs.

You tend to skirt your object placements around the edges of the room, rather than having many things that protrude into the central spaces, which causes you to end up with spaces like this
image
This room is a good shape, size, has the bathroom there, it has all the elements it needs, however it has this large hole of space in the middle of the room. You could make the couch bigger, add a table, add vendors. But you need to use up more space.

Your Cryo room has a similar, but smaller issue.
image
You have this table here that has nothing on it, with an almost entirely empty 3x3.

I'm actually a bit sad to make a fuss about this part, because I think it looks really nice, however the port on a ship that allows a subshuttle to dock to it, needs to be able to be air sealed once the ship docks. This will not do that.
image

This room, while I really enjoy the layout, feels like a showroom showing how you could arrange furniture, rather than an actually used space. Add things to the tables, add some clutter to the room, and do something with the 4x3 space to the left of the kitchen.
image

This is probably the biggest offender on the ship in terms of being too empty. You have a lot of unused space.
image

These engine rooms are a great start, but again, just need more stuff in them. The top nacelle is better than the bottom in that regard, but still needs more clutter.
image
image

Same with the bridge.
image

Overall you have an extremely strong start. I really enjoy how you've layed out your rooms, I love the sillhouette, and I think once you get the ship a bit more filled in, this will be a great ship.

**Comment by thearbiber** [comment]
Will continue work on this Soon, a bit busier IRL now. Thank you for the feedback though, it is appreciated.

**Pull request closed without merging**